app/src/main/res/layout/fragment_home.xml

```diff
@@ -0,0 +1,235 @@
+        android:layout_marginEnd="@dimen/dp_8"
+        android:layout_marginStart="@dimen/dp_8"
+        android:layout_marginTop="@dimen/dp_8"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textDaily"
+        >
+
+    </android.support.v7.widget.RecyclerView>
+
+    <TableLayout
+        android:id="@+id/tableDetail"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="@dimen/dp_8"
+        android:layout_marginStart="@dimen/dp_8"
+        android:layout_marginTop="@dimen/dp_8"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/recyclerviewDaily">
+
+        <TableRow>
+
+            <ImageView
+                android:layout_width="@dimen/dp_50"
+                android:layout_height="@dimen/dp_50"
+                android:background="@drawable/ic_temperature"
+                />
+
+            <TextView
+                android:id="@+id/textTempDetail"
+                android:layout_width="@dimen/dp_50"
+                android:layout_height="@dimen/dp_50"
+                android:gravity="center"
+                android:textColor="@color/color_black"
+                android:textSize="@dimen/sp_25"
+                />
+
+            <ImageView
+                android:layout_width="@dimen/dp_50"
+                android:layout_height="@dimen/dp_50"
+                android:background="@drawable/ic_wind"
+                />
+
+            <TextView
+                android:id="@+id/textWindDetail"
+                android:layout_width="@dimen/dp_50"
+                android:layout_height="@dimen/dp_50"
+                android:gravity="center"
+                android:textColor="@color/color_black"
+                android:textSize="@dimen/sp_25"
+                />
+
+        </TableRow>
+        <TableRow>
+
+            <ImageView
+                android:layout_width="@dimen/dp_50"
+                android:layout_height="@dimen/dp_50"
+                android:background="@drawable/ic_drop"
+                />
+
+            <TextView
+                android:id="@+id/textHumidityDetail"
+                android:layout_width="@dimen/dp_50"
+                android:layout_height="@dimen/dp_50"
+                android:gravity="center"
+                android:textColor="@color/color_black"
+                android:textSize="@dimen/sp_25"
+                />
+
+            <ImageView
+                android:layout_width="@dimen/dp_50"
+                android:layout_height="@dimen/dp_50"
+                android:background="@drawable/ic_cloud"
+                />
+
+            <TextView
+                android:id="@+id/textCloudDetail"
+                android:layout_width="@dimen/dp_50"
+                android:layout_height="@dimen/dp_50"
```

Comment:
việc set cứng toàn bộ kích thước view đồng nghĩa với việc view ứng dụng của e chỉ chạy ổn trên 1 kích thước màn hình duy nhất. 
khi ở màn hình khác đa số tất cả đều đi bể view hết e